tests/iam_policies/cluster-roles.cfn.yaml

```diff
@@ -26,7 +26,7 @@ Resources:
                   - logs:CreateLogStream
                   - logs:PutLogEvents
                 Effect: Allow
-                Resource: !Sub arn:${AWS::Partition}:logs:*:*:*
+                Resource: !Sub arn:${AWS::Partition}:logs:${AWS::Region}:${AWS::AccountId}:log-group:/aws/parallelcluster/*
```

Comment:
I think this is wrong cause lambda does not write logs to `/aws/parallelcluster/*`

Reply:
Fixed: https://github.com/aws/aws-parallelcluster/pull/3678/commits/816041457ae921e851f881c36f4186b1e0517538